app/client/cli/utils.go

```diff
@@ -0,0 +1,145 @@
+package cli
+
+import (
+	"bufio"
+	"bytes"
+	"context"
+	"encoding/json"
+	"fmt"
+	"io"
+	"log"
+	"math/rand"
+	"net/http"
+	"os"
+	"strings"
+	"time"
+
+	"github.com/pokt-network/pocket/app/pocket/rpc"
+	"github.com/pokt-network/pocket/shared/codec"
+	"github.com/pokt-network/pocket/shared/crypto"
+	utilityTypes "github.com/pokt-network/pocket/utility/types"
+	"golang.org/x/crypto/ssh/terminal"
+)
+
+// readEd25519PrivateKeyFromFile returns an Ed25519PrivateKey from a file where the file simply encodes it in a string (for now)
+// HACK(pocket/issues/150): this is a temporary hack since we don't have yet a keybase, the next step would be to read from an "ArmoredJson" like in V0
+func readEd25519PrivateKeyFromFile(pkPath string) (pk crypto.Ed25519PrivateKey, err error) {
+	pkFile, err := os.Open(pkPath)
+	if err != nil {
+		return
+	}
+	defer pkFile.Close()
+	pk, err = parseEd25519PrivateKeyFromReader(pkFile)
+	return
+}
+
+func parseEd25519PrivateKeyFromReader(reader io.Reader) (pk crypto.Ed25519PrivateKey, err error) {
+	if reader == nil {
+		return nil, fmt.Errorf("cannot read from reader %v", reader)
+	}
+
+	buf := new(bytes.Buffer)
+	buf.ReadFrom(reader)
+
+	priv := &crypto.Ed25519PrivateKey{}
+	err = priv.UnmarshalJSON(buf.Bytes())
+	if err != nil {
+		return
+	}
+	pk = priv.Bytes()
+	return
+}
+
+// Credentials reads a password from the prompt and returns the trimmed version
+//
+// If pwd is provided (via flag to the command), it uses that one instead of asking via prompt
+func Credentials(pwd string) string {
```

Comment:
Reduce function scope if we don't need it outside of this package.

Reply:
Addressed in https://github.com/pokt-network/pocket/pull/177/commits/8aef5479a0d57dc1bfda6436c106a4ef0876d755